drivers/sensor/hs300x/hs300x.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023 Ian Morris
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_HS300X_HS300X_H_
+#define ZEPHYR_DRIVERS_SENSOR_HS300X_HS300X_H_
+
+#include <zephyr/device.h>
+
+struct hs300x_config {
+	struct i2c_dt_spec bus;
+};
+
+struct hs300x_data {
+	int16_t t_sample;
+	uint16_t rh_sample;
+};
```

Comment:
Not enough here to justify having a separate private include file. Please move into `hs300x.c`

Reply:
done